NETWORK/NetworkAddEntityToSynchronisedScene.md

```diff
@@ -5,16 +5,42 @@ ns: NETWORK
 
 ```c
 // 0xF2404D68CBC855FA 0x10DD636C
-void NETWORK_ADD_ENTITY_TO_SYNCHRONISED_SCENE(Entity entity, int netScene, char* animDict, char* animName, float speed, float speedMulitiplier, int flag);
+void NETWORK_ADD_ENTITY_TO_SYNCHRONISED_SCENE(Entity entity, int netScene, char* animDict, char* animName, float blendInSpeed, float blendOutSpeed, float flag);
 ```
 
 
 ## Parameters
-* **entity**: 
-* **netScene**: 
-* **animDict**: 
-* **animName**: 
-* **speed**: 
-* **speedMulitiplier**: 
-* **flag**: 
+* **entity**: The Entity that the animation is gonna play on (Recomended that the entity is serversided for best syncing).
+* **netScene**: scene id.
+* **animDict**: The animDict of the animation that you want to play (remember you have to load with RequestAnimDict() before executing this native)
+* **animName**: The name of the animation that the entity is gonna play.
+* **blendInSpeed**: a float (normal speed is 8.0f).
+* **blendOutSpeed**: a float (normal speed is -8.0f).
+* **flag**: A float 1000.0f (Also doesn't seem to affect anything).
 
+## Animation Dictionaries
```

Comment:
You have to use triple hashes for this, otherwise there will be compile errors.
For example:

`### Animation Dictionaries`